packages/packages/v1-adapters/src/dispatchers/__tests__/index.test.ts

```diff
@@ -0,0 +1,115 @@
+import { openRoute, runCommand } from '../';
+
+describe( '@elementor/v1-adapters/dispatchers', () => {
+	beforeEach( () => {
+		( window as any ).$e = {
```

Comment:
Consider defining one reusable variable.

Reply:
for `window as any`?
same as above, maybe it'll have references issues